prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -225,4 +237,42 @@ object Metadata {
             else -> error("too many sub-names")
         }
     }
-}
\ No newline at end of file
+
+    /*
```

Comment:
In a way, but I think we should keep both. ValueSet can have metadata like description that is needed. ValueSet is all over HL7 messages, so it is a useful abstraction. There only appears to be one table that we need in the COVID topic, the LIVD table. The table doesn't have metadata because it is a csv. We could define our own format, but that means translating the LIVD table. So, I'm keeping both for now.